packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,89 @@
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-input-focus-background-start, 0 0);
+    border: var(--terra-form-input-border, 1px solid #4a4a4a);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    box-shadow: var(--terra-form-input-box-shadow, none);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.357rem);
+    padding-right: var(--terra-form-input-padding-right, 0.357rem);
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border: var(--terra-form-input-hover-border, 1px solid #4a4a4a);
+    }
+
+    &[required] {
+      box-shadow: none;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border: var(--terra-form-input-disabled-border, 1px solid #dedfe0);
+      color: var(--terra-form-input-disabled-color, rgba(0, 0, 0, 0.25));
+      cursor: default;
+      opacity: var(--terra-form-input-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-focus-background-color, #fff);
+      background-size: var(--terra-form-input-focus-keyboard-animation-size, 0 0);
+      border: var(--terra-form-input-focus-border, 1px solid #4a4a4a);
+      opacity: var(--terra-form-input-focus-opacity, 1);
+      outline: none;
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+      }
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
+      }
+    }
+  }
+
+  .form-error {
+    background-image: var(--terra-form-input-error-focus-animation-background, none);
+    border: var(--terra-form-input-error-border, 1px solid #ff0b0f);
+    opacity: var(--terra-form-input-error-opacity, 1);
+
+    &:hover {
+      border: var(--terra-form-input-error-hover-border, 1px solid #ff0b0f);
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-error-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border: var(--terra-form-input-error-disabled-border, 1px solid #ff0b0f);
+      color: var(--terra-form-input-error-disabled-color, rgba(0, 0, 0, 0.25));
+      opacity: var(--terra-form-input-error-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-error-focus-background-color, #fff);
```

Comment:
Again, should this be white or transparent?

Reply:
My callouts had this listed as white. It's also listed as white [here](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.scss#L3) which I was told was the example I should be following.

Reply:
#fff (white) for default theme